services/utils/objectToLabelValueArray.js

```diff
@@ -0,0 +1,5 @@
+export const objectToLabelValueArray = (object) => {
+    return Object.entries(object).map(([key, value]) => {
+        return {label: value, value: key}
+    });
+}
```

Comment:
Formatting on this file is incorrect

```suggestion
export const objectToLabelValueArray = (object) => {
  return Object.entries(object).map(([key, value]) => {
    return { label: value, value: key };
  });
};

```